src/types/LoadStatuses.ts

```diff
@@ -0,0 +1,30 @@
+export interface Weather {
+       main: { 
+         temp: number; 
+         feels_like: number;
+         humidity:number;
+         pressure: number;
+       };
+       wind: { speed: number };
+       id?: number;
+       weather: {icon: string}[];
+     }
+
+export enum LOAD_STATUSES_TYPES {
+       SET_LOADING = "loading",
+       SET_ERROR = "error",
```

Comment:
добавляем префекс в виде названия слайса, чтобы когда будет 2+ слайса не было конфликтов (одинаковых названий экшенов). Сейчас у тебя 1 слайс, но лучше сразу добавить, чем потом переделывать или забывать переделать и баг словить
```suggestion
              SET_ERROR = "weather/error",
```